frontend/javascripts/dashboard/dataset_view.js

```diff
@@ -187,6 +201,68 @@ function DatasetView(props: Props) {
     );
   }
 
+  function renderNewJobsAlert() {
+    const now = moment();
+    const newJobs = jobs
+      .filter(
+        job =>
+          job.type === "convert_to_wkw" && moment.duration(now.diff(job.createdAt)).asDays() <= 3,
+      )
+      .sort((a, b) => b.createdAt - a.createdAt);
+
+    if (newJobs.length === 0) {
+      return null;
+    }
+
+    const newJobsHeader = (
+      <React.Fragment>
+        Recent Dataset Conversions{" "}
+        <Tooltip
+          title="The conversion of the displayed datasets were started in the last 3 days."
+          placement="right"
+        >
+          <InfoCircleOutlined />
+        </Tooltip>
+      </React.Fragment>
+    );
+    const newJobsList = (
+      <div style={{ paddingTop: 8 }}>
+        {newJobs.slice(0, 5).map(job => {
+          const { tooltip, icon } = TOOLTIP_MESSAGES_AND_ICONS[job.state];
+          return (
+            <Row key={job.id} gutter={16}>
+              <Col span={10}>
+                <Tooltip title={tooltip}>{icon}</Tooltip>
+                {` ${job.datasetName || "UNKNOWN"}`}
+                &nbsp;(started at&nbsp;
```

Comment:
Could you please add the non-breaking space (`&nbsp;`) as a constant? Just like it is done with `ThinSpace`? This would add a name for this arbitrary value that explains it better. 

According to [this post](https://stackoverflow.com/questions/24432576/reactjs-render-string-with-non-breaking-spaces/36804422) `"\u00a0"` should be the utf8 equivalent.

And while you are at it, could you please also use the newly created variable for all other occurrences of `&nbsp;`? :slightly_smiling_face:

Reply:
good idea :+1: